### DISCUSSION TITLE
[Project] Allow comments in Drafts

### DISCUSSION THREAD

**Discussion by infinite-persistence** [community/community#33113] [Projects and Issues]
Would be nice if Drafts can support comments to avoid having to discuss Drafts in Slack.
Also, Drafts don't seem to have a specific URL that I can reference to in external chat, so an internal commenting system would be great.

**Comment by jfullstackdev** [comment]
I'm not very sure of your situation but if you need a group chat, you can have a private repo, add them, then enable GitHub Discussions and you are all set
